integ-test/src/test/java/org/opensearch/sql/legacy/DateFunctionsIT.java

```diff
@@ -44,271 +33,205 @@ protected void init() throws Exception {
   }
 
   @Test
-  public void year() throws IOException {
-    SearchHit[] hits = query(
-        "SELECT YEAR(insert_time) as year"
-    );
-    for (SearchHit hit : hits) {
-      int year = (int) getField(hit, "year");
-      DateTime insertTime = getDateFromSource(hit, "insert_time");
-      assertThat(year, equalTo(insertTime.year().get()));
-    }
+  public void year() {
+    String query = "SELECT YEAR(insert_time) as year " + FROM;
+    JSONObject result = executeJdbcRequest(query);
+    assertEquals(9936, result.getInt("total"));
+    verifySchema(result, schema("YEAR(insert_time)", "year", "integer"));
+    assertEquals(2014, result.getJSONArray("datarows").getJSONArray(0).getInt(0));
```

Comment:
We are only checking the first row.  Can we check all the rows like the previous test?

Reply:
Total amount of rows returned by this query is 9936, and all must contain the same data, so checking just one is sufficient in my opinion.
<img width="915" alt="Screenshot 2023-01-25 at 3 56 33 PM" src="https://user-images.githubusercontent.com/116052492/214720918-c82cb07d-6bfd-4838-a2f3-89d5971bb784.png">
<img width="915" alt="Screenshot 2023-01-25 at 3 56 33 PM" src="https://user-images.githubusercontent.com/116052492/214720918-c82cb07d-6bfd-4838-a2f3-89d5971bb784.png">

Reply:
If you wanna check rows without checking every row you can add a `LIMIT` of something like 5

Reply:
Yes, but I'm not sure I am supposed to modify the queries anyhow. So far I've never touched any query during the refactoring.